sections/discussion/ourprocessreflections.tex

```diff
@@ -0,0 +1,78 @@
+If two people are in process of reviewing a pull request another person can be waiting for them to finish if they are assigned a pull request with one of them. 
+Then when a pull request had been reviewed and if changes are requested, then these people had to come back to review the changes.
+To make this more organized a we created a PR spreadsheet to give an better overview of which pull requests you are assigned to, and who are currently requested to do many or few reviews.
+This can be seen on \autoref{fig:formal-reviews}.
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=\linewidth]{figures/formal-reviews.PNG}
+    \caption{Pull request assignment sheet.}
+    \label{fig:formal-reviews}
+\end{figure}
+
+\subsubsection{Working from home}
+Unlike other semesters, we were not assigned a group room this semester.
+This resulted in us working from home most days in the week, because we thought this would be easier and as productive as if we met at the university.
+One of the cons of working from home is that it is more difficult to get help from the other group members and this can lead people to get stuck on tasks.
+After a few weeks we realized that it would be better if we had one workday together every week.
+This was really beneficial as the more experienced React developers were able to help other members that were less experienced.
+It was also easier to conduct a sprint planning when we met in person.
+
+\subsubsection{Supervisor meetings}
+The day before the supervisor meeting we sent the report, a reading guide and an agenda for the meeting.
+Our supervisor gave feedback on the material that we sent to her and during the meeting we presented the things that we had done the previous week.
+These meetings have been longer than the meetings we had previously experienced in last semesters, but this also resulted in better feedback, as the questions she asked made us reflect on certain choices we had made.
+
+\subsubsection{What we would do differently}
+One of the problems that we faced was that people could get stuck on tasks.
+For future projects we would try to be more aware of where people are in the process, so that it would be easier to help them with solving their tasks.
+One of the solutions to this could be daily standups, so that we are aware of where people are with their tasks, and which once they have not completed.
+
+Another solution could be to have more workdays at the university, so that people have an easier time getting help from one another.
+It is especially important in the beginning of the semester when there are still a lot of ambiguities and uncertainties with the project.  
+
+We would also like to hold the retrospective meeting so that we had to reflect on the previous week and correcting the problems that we faced.
+
+When adding tasks to the product backlog it should also be a requirement to add a DoD, as we multiple times were unable to recognize what needed to be done from just the title.
```

Comment:
```suggestion
When adding tasks to the product backlog it should also be a requirement to add a DoD, since we experienced being unable to recognize what needed to be done from just the title.
```